src/Compatibility/Microsoft.DotNet.ApiCompatibility/DefaultSymbolsEqualityComparer.cs

```diff
@@ -24,7 +24,8 @@ symbol switch
                 IPropertySymbol => symbol.ToComparisonDisplayString(),
                 IEventSymbol => symbol.ToComparisonDisplayString(),
                 ITypeSymbol => symbol.ToComparisonDisplayString(),
-                _ => symbol.Name,
+                INamespaceSymbol => symbol.ToComparisonDisplayString(),
+                _ => symbol.ToComparisonDisplayString(),
```

Comment:
Did you mean to change the default case?  Does this mean we just always call `ToComparisonDisplayString()` and the switch does nothing?

Reply:
Actually, if I'm understanding this correctly as an identifier we use as a key for comparison, shouldn't we use DocID?

Reply:
yes we can remove the switch here. I think DisplayString is pretty close to documentation ID as well. its returning the strings in same manner

Reply:
Does it include the prefix to indicate the symbol type (eg property/field/type/etc)?

Reply:
No it doesnt include that yet.

Reply:
I see, I wonder what other sorts of ambiguities might exist if we are omitting those designations cc @safern